packages/grpc-js-xds/interop/Dockerfile

```diff
@@ -27,7 +27,7 @@ RUN npm install
 WORKDIR /node/src/grpc-node/packages/grpc-js-xds
 RUN npm install
 
-FROM node:18-slim
+FROM gcr.io/distroless/nodejs18-debian11:latest
```

Comment:
I think it expects `/nodejs/bin/node` instead of `node` in ENTRYPOINT: https://github.com/GoogleContainerTools/distroless/blob/main/nodejs/BUILD#L38